dali/pipeline/operators/reader/loader/video_loader.h

```diff
@@ -135,9 +139,11 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
       stop_(false) {
     if (step_ < 0)
       step_ = count_ * stride_;
+    channels_ = 3;
```

Comment:
Is it used anywhere?

Reply:
It can be hardcoded, its only used once
https://github.com/NVIDIA/DALI/blob/67997f5fb10b04d4581a66bf5e8f66d4bd3ee24f/dali/pipeline/operators/reader/loader/video_loader.cc#L505

Reply:
So how about some const expr or something?